### DISCUSSION TITLE
Getting an error

### DISCUSSION THREAD

**Discussion by imbakar** [spatie/laravel-backup#1122] [Q&A]
Hello,
Hope you are doing well.

Backup failed because: The Process class relies on proc_open, which is not available on your PHP installation..

Thanks

**Comment by freekmurze** [comment]
Make sure it is available.

**Comment by imbakar** [comment]
Hello,
Thanks for your quick response, I am on a shared cpanel hosting & maybe the `proc_open` is disable.

Also followed below salutation not working

```
php artisan vendor:publish --tag=flare-config
```

```
'reporting' => [
        'anonymize_ips' => true,
        'collect_git_information' => false,
        'report_queries' => true,
        'maximum_number_of_collected_queries' => 200,
        'report_query_bindings' => true,
        'report_view_data' => true,
    ],
```

Is there any option to use it without `proc_open`

Thanks
